jsconfig.json

```diff
@@ -1,14 +1,3 @@
 {
-  "exclude": [
-    "*.log",
-    "builds",
-    "coverage",
-    "dist",
-    "docs",
-    "lavamoat",
-    "node:console",
-    "node_modules",
-    "patches",
-    "test-artifacts"
-  ]
```

Comment:
were these exclusions no needed here or?

Reply:
oh, not part of this i'll remove. With them in autocomplete just randomly starts failing for me.